src/rollup/RollupCreator.sol

```diff
@@ -98,6 +103,8 @@ contract RollupCreator is Ownable {
             osp
         );
 
+        address tempOwner = config.owner;
+        config.owner = address(this);
```

Comment:
confusing name, maybe `actualOwner` instead of `tempOwner`? also add some comment e.g.
```suggestion
        // initialize the rollup with this contract as owner to set batch poster and validators
        // it will transfer the ownership back to the actual owner later
        address actualOwner = config.owner;
        config.owner = address(this);
```